main.py

```diff
@@ -0,0 +1,83 @@
+import os
+from typing import Any, Dict
+
+from dotenv import load_dotenv
+from fastapi import FastAPI, HTTPException
+from openai import OpenAI
+from pydantic import BaseModel
+
+from pseudoanonymize.anonymization import Anonymizer
+from pseudoanonymize.deanonymization import Deanonymizer
+from pseudoanonymize.exceptions import MaxRetriesExceededException
+
+app = FastAPI()
+
+load_dotenv()
+openai_api_key = os.getenv("OPENAI_API_KEY")
+client = OpenAI(api_key=openai_api_key)
+
```

Comment:
Having the whole service in a single file is fine for now as it's very small but if the service grows then I'd suggest splitting up the code into several files.

The folder structure I typically use for FastAPI projects is e.g."

```
> models
    > user.py  <-- Core Pydantic models that are shared across endpoints
> endpoints
    > user_create.py <-- One endpoint per file
    > user_read.py
> main.py  <-- there FastAPI service itself
```

If you want you can put this structure in place now but not required.

Reply:
This came to my mind when designing the code base, however, I tried to avoid it since it felt like overengineering. Since as you mentioned this service is simple in terms of endpoints/models. So maybe it makes sense to keep it this way for now and refactor it later once things grow?

Reply:
Agree - for now it's absolutely fine as it is 👍